_recr_variability.tex

```diff
@@ -5,25 +5,25 @@
 
 \begin{figure}[H]
 	\begin{center}
-		\includegraphics[scale = 0.70]{appendixA_recruits}\\
+		\includegraphics[alt={Spawner-recruitment relationship for darkblotched rockfish \citep{gertseva-status-2013}. Red points represent estimated recruitments, the solid black line is the stock-recruit relationship and the green line represents the adjustment to this relationship after adjustment to account for the log-normal distribution associated with each year. The ``+'' symbol labeled 1915 near the right side represents both the virgin and initial equilibrium of the model. The numerous red points close to the initial conditions correspond to the early years of the model with low harvest rates.},scale = 0.70]{appendixA_recruits}\\
```

Comment:
alternative text will not render things like citations to the best of my knowledge but I might be wrong.

Reply:
Also, the last time that I checked commas are not valid in alternative text unless the entire text is wrapped in quotations.